core/resources/shaders/polyline.vs

```diff
@@ -35,6 +36,18 @@ varying vec2 v_texcoord;
     varying vec4 v_lighting;
 #endif
 
+vec4 modelPosition() {
+    return a_position;
+}
+
+vec4 worldPosition() {
+    vec4 worldPosition = a_position * u_model + vec4(u_map_position.xy, 0., 1.);
```

Comment:
Ohhh good one, I can see how that would muck up procedural textures. I'll see what I can do about that...